tck/features/precategorized/operations/patterns/Pattern1.feature

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (c) 2015-2020 "Neo Technology,"
+# Network Engine for Objects in Lund AB [http://neotechnology.com]
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Attribution Notice under the terms of the Apache License 2.0
+#
+# This work was created by the collective efforts of the openCypher community.
+# Without limiting the terms of Section 6, any Derivative Work that is not
+# approved by the public consensus process of the openCypher Implementers Group
+# should not be described as “Cypher” (and Cypher® is a registered trademark of
+# Neo4j Inc.) or as "openCypher". Extensions by implementers or prototypes or
+# proposals for change that have been documented or implemented should only be
+# described as "implementation extensions to Cypher" or as "proposed changes to
+# Cypher that are not yet approved by the openCypher community".
+#
+
+#encoding: utf-8
+
+Feature: Pattern1 - Compute the number of subgraphs induced by a pattern expression
+
+  Scenario: Functions should return null if they get path containing unbound
```

Comment:
This is a very weird scenarios. Hard to place. Probably need to be rip up. 

I like putting something in `precategorized` to not through away work. However, `Pattern1` look wrong. Two of the functions are path functions (`nodes`, `relationships`), one is a graph function (`types`), and one is a list functions (`size`), non is a pattern function. So, I guess, I would leave this scenario in `uncategorized/FunctionsAcceptance.feature` for now and deal with it in the final clean up round of categorization, when we will probably rip it in four pieces. 

You could add a comment to the scenario like "Consider splitting into one scenario per one of four function:`nodes`, `relationships`, `types`, and `size`." to keep record of this intension.

Reply:
restored to `uncategorized/FunctionsAcceptance.feature`. Comment added